cmake/MacOSBundle.cmake

```diff
@@ -0,0 +1,29 @@
+function(set_bundle_properties executable_name)
+    if(${executable_name} STREQUAL "mrview")
```

Comment:
mrview should not only include the document-icon, but also the icon for the application itself (like shview does)

Reply:
Done.